src/client/pages/OfferNew/utils.ts

```diff
@@ -61,6 +61,17 @@ export const getOfferData = (quoteBundle: QuoteBundle): OfferData => {
   }
 }
 
+const getHouseholdSizeFromBundledQuotes = (quotes: BundledQuote[]): number => {
+  const quoteDetails = quotes.map((x) => x.quoteDetails)
+
+  for (const details of quoteDetails) {
+    if ('householdSize' in details) return details.householdSize
+    if ('coInsured' in details) return details.coInsured + 1
+  }
+
+  return 0
```

Comment:
I think it might not be a great idea to return `0` as some sort of default. If we neither have `householdSize` nor `coInsured` on the `quoteDetails` something is seriously wrong and that should probably be communicated.

Reply:
Same idea as the previous comment. Let's discuss this further.